lib/components/ReactBingmaps/ReactBingmaps.js

```diff
@@ -107,9 +107,12 @@ var ReactBingmaps = function (_Component) {
 		key: 'componentWillReceiveProps',
 		value: function componentWillReceiveProps(nextProps) {
 			var mapReference = nextProps.id ? '#' + nextProps.id : '.react-bingmaps';
-			if (this.props.center.join() !== nextProps.center.join()) {
+			if (this.props.center && this.props.center.join() !== nextProps.center.join()) {
 				this.setMapCenter(nextProps.center, mapReference);
 			}
+			if (this.props.bounds && this.props.bounds.join() !== nextProps.bounds.join()) {
+				this.setMapBounds(nextProps.bounds, mapReference);
+			}
```

Comment:
You also need to compile this and commit the result.  (See my previous commits).  The repo's docs show the command you need to run to generate the vanillaJS code